.npmignore

```diff
@@ -1 +0,0 @@
-bench
```

Comment:
We need only add folder `test`.
But ok, I'll change back or we create new holywar thread about files/npmignore :grinning: